### PR TITLE
Remove redundant makefiles

### PR DESCRIPTION
**Pull request by jamierocks**
As far as I know nobody uses them, and because of the Gulp scripts would be silly to

### CODE REVIEW

**Comment by Tzky** [comment]
When building the docs on travis, it doesn't make use of gulp but rather builds it via the scripts located in ``/etc/``. I'm not 100% sure if Sphinx uses the makefile scripts when we invoke a build via travis ;)

**Comment by jamierocks** [comment]
The Makefile executes `sphinx-build`.

File: Gulpfile.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ gulp.task('webserver', ['sphinx'], function () {
 });
 
 gulp.task('watch', function () {
-    gulp.watch(['source/**/*'], ['sphinx']);
+    gulp.watch('./source/**/*', 'sphinx');
```
Comment (Minecrell) [inline Gulpfile.js]: `gulp.watch` doesn't accept a single string as the tasks to execute, this change breaks the `watch` task:

```
TypeError: Cannot create property 'mark' on string 'sphinx'
    at new Gaze (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/gaze/lib/gaze.js:41:13)
    at gaze (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/gaze/lib/gaze.js:86:10)
    at Object.module.exports [as watch] (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/glob-watcher/index.js:12:17)
    at Gulp.watch (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/gulp/index.js:40:14)
    at Gulp.<anonymous> (/home/minecrell/dev/minecraft/sponge/SpongeDocs/Gulpfile.js:28:10)
    at module.exports (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/home/minecrell/dev/minecraft/sponge/SpongeDocs/node_modules/orchestrator/index.js:134:8)
    at /usr/lib/node_modules/gulp/bin/gulp.js:129:20
```

Comment (jamierocks) [inline Gulpfile.js]: thats odd **i think** it worked for me when i tested :s

Comment (jamierocks) [inline Gulpfile.js]: resolved

**Pull request merged**